packages/xo-server-web-hooks/src/index.js

```diff
@@ -139,6 +155,11 @@ export const configurationSchema = ({ xo: { apiMethods } }) => ({
             title: 'URL',
             type: 'string',
           },
+          response: {
+            description: 'Waiting for the server response before execute the job. Only available on "PRE" type',
+            title: 'Response',
```

Comment:
"Wait for response"? "Response" sounds like you need to provide a response, somehow.